packages/graphql-cohttp/graphql-cohttp.0.13.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "Andreas Garnaes <andreas.garnaes@gmail.com>"
+authors: "Andreas Garnaes <andreas.garnaes@gmail.com>"
+homepage: "https://github.com/andreas/ocaml-graphql-server"
+doc: "https://andreas.github.io/ocaml-graphql-server/"
+bug-reports: "https://github.com/andreas/ocaml-graphql-server/issues"
+dev-repo: "git+https://github.com/andreas/ocaml-graphql-server.git"
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.1"}
+  "graphql" {>= "0.9.0"}
+  "cohttp" {>= "2.0.0"}
+  "crunch"
+  "astring" {>= "0.8.3"}
+  "base64" {>= "3.0.0"}
+  "ocplib-endian" {>= "1.0"}
+  "digestif" {>= "0.7.0"}
+  "alcotest" {with-test}
+  "cohttp-lwt-unix" {with-test}
+  "graphql-lwt" {with-test}
```

Comment:
There is no version of `graphql-lwt` compatible with `graphql.0.13.0` according to this failure: https://ci.ocaml.org/log/saved/docker-build-a4e3cf40ef2d7329ec2cd3c77de275c8/db15c461e74eae4cbaeb0669cf862e387788292f